test/fetch-test-deps.sh

```diff
@@ -1,10 +1,46 @@
 #!/usr/bin/env bash
-
 set -e
 
 cd "$(dirname "$0")"
 
-case "$1" in
+# Parse arguments and provide usage message
+function usage {
+	echo "Downloads source code of Game Boy programs used as RGBDS test cases."
+	echo "Options:"
+	echo "    -h, --help      show this help message"
+	echo "    --dfsg          download only freely licensed programs"
```

Comment:
I'd prefer `--only-free` then. The double negative is because I felt "nonfree" would be less ambiguous than "free" (the usual beer vs. freedom thing).

Reply:
Use of `--dfsg` was inspired by [use of `+dfsg.1` in package version numbers](https://www.reddit.com/r/debian/comments/66094l/what_is_dfsg_in_package_version_numbers/) when something non-free from the upstream repository is cut out.

Reply:
Understandable, but I fear that acronym is more niche than our target user base—it makes sense for package versions set by Debian for Debian, less so for us.

I prefer flag names that are more self-describing, hence "[set this to require] no nonfree".